.github/workflows/build_wheels.yml

```diff
@@ -169,6 +171,7 @@ jobs:
           go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.26.0
           go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.1.0
           pip install dist/*tar.gz
+      # py3.10 on MacOS does not work with Go so we have to install separately.
```

Comment:
What's the specific failure? Can you make an issue on github with more context and link to it here?

Reply:
fixed